### ISSUE TITLE
ImportError: cannot import name 'V1alpha2TensorRTSpec' from 'kfserving' 

### ISSUE THREAD

**Issue by hamedhsn** [kubeflow/fairing#521]
/kind bug

**What steps did you take and what happened:**
trying to update my dependencies and seems that underlying kfserving package introduced this bug in new version. 
```    from kubeflow import fairing
/etc/pyenv/versions/3.7.5/lib/python3.7/site-packages/kubeflow/fairing/__init__.py:7: in <module>
    from kubeflow.fairing.config import config
/etc/pyenv/versions/3.7.5/lib/python3.7/site-packages/kubeflow/fairing/config.py:16: in <module>
    from kubeflow.fairing.deployers.kfserving.kfserving import KFServing
/etc/pyenv/versions/3.7.5/lib/python3.7/site-packages/kubeflow/fairing/deployers/kfserving/kfserving.py:26: in <module>
    from kfserving import V1alpha2TensorRTSpec
E   ImportError: cannot import name 'V1alpha2TensorRTSpec' from 'kfserving' (/etc/pyenv/versions/3.7.5/lib/python3.7/site-packages/kfserving/__init__.py)

```

reverting back to `kfserving = "0.3.0.1"` solve the issue but would be good if that is fixed.
I know @jinchihe you are involved with kfserving can you please have a look at it. should be a quick fix. 

Thanks!

**Comment by jinchihe** [comment]
@hamedhsn I'm going to fix this in PR #522 Thanks.

**Comment by jinchihe** [comment]
The problem already fixed in #522, and we can try the cases with new kubelow-fairing SDK v1.0.1. Thanks

/close
